fineract-provider/src/main/java/org/apache/fineract/portfolio/client/domain/ClientStatus.java

```diff
@@ -18,17 +18,17 @@
  */
 package org.apache.fineract.portfolio.client.domain;
 
+import org.springframework.util.StringUtils;
+
 /**
  * Enum representation of client status states.
  */
 public enum ClientStatus {
 
-    INVALID(0, "clientStatusType.invalid"), //
-    PENDING(100, "clientStatusType.pending"), //
-    ACTIVE(300, "clientStatusType.active"), //
-    TRANSFER_IN_PROGRESS(303, "clientStatusType.transfer.in.progress"), //
-    TRANSFER_ON_HOLD(304, "clientStatusType.transfer.on.hold"), //
-    CLOSED(600, "clientStatusType.closed"), REJECTED(700, "clientStatusType.rejected"), WITHDRAWN(800, "clientStatusType.withdraw");
+    INVALID(0, "clientStatusType.invalid"), PENDING(100, "clientStatusType.pending"), ACTIVE(300,
+            "clientStatusType.active"), TRANSFER_IN_PROGRESS(303, "clientStatusType.transfer.in.progress"), TRANSFER_ON_HOLD(304,
+                    "clientStatusType.transfer.on.hold"), CLOSED(600, "clientStatusType.closed"), REJECTED(700,
+                            "clientStatusType.rejected"), WITHDRAWN(800, "clientStatusType.withdraw");
 
```

Comment:
Hmm... couldn't spot any difference except removal of the formatting (one status per line) - was there something changed here?

Reply:
I removed the useless comments, that made it happen. 
Weird, we might need to change some spotless rules.